src/Composer/Installers/DrupalInstaller.php

```diff
@@ -14,5 +14,7 @@ class DrupalInstaller extends BaseInstaller
         'custom-module'    => 'modules/custom/{$name}/',
         'custom-profile'   => 'profiles/custom/{$name}/',
         'drupal-multisite' => 'sites/{$name}/'
```

Comment:
Missing trailing `,` here

Reply:
@alcohol Thank for your review, I fixed the issue.